site/cds_rdm/inspire_harvester/writer.py

```diff
@@ -30,81 +32,126 @@ def _write_entry(self, entry, *args, **kwargs):
 
         existing_records_hits = existing_records.to_dict()["hits"]["hits"]
         existing_records_ids = [hit["id"] for hit in existing_records_hits]
+        current_app.logger.debug(
+            f"IDs of existing records found: {existing_records_ids}."
+        )
 
         if multiple_records_found:
+            current_app.logger.error(
```

Comment:
for the line below, I think this would be rather a partial success exception, it shouldn't stop other records from being imported